backend/src/models/CategoryManager.js

```diff
@@ -0,0 +1,13 @@
+const AbstractManager = require("./AbstractManager");
+
+class CategoryManager extends AbstractManager {
+  constructor() {
+    super({ table: "category" });
+  }
+
+  findAllByTitle() {
+    return this.database.query(`SELECT title FROM ${this.table}`);
```

Comment:
You are not filtering by title here 👀

Reply:
If you wanted to get element depend on a title, modify this sql request. But if you just want to get title column of all your categories, name of your functions is not consistent.